[documentation] Add trouble shooting step to the install doc (#5336)
MystenLabs/sui: doc/src/build/install.md

* [`sui`](cli-client.md) - The Sui CLI tool contains subcommands for enabling `genesis` of validators and accounts, starting the Sui network, and [building and testing Move packages](move/index.md), as well as a [client](cli-client.md) for interacting with the Sui network.
* [`rpc-server`](json-rpc.md) - run a local Sui gateway service accessible via an RPC interface.

Trouble shooting:

> If the previous command fails, try to make sure your Rust version is up to date:

Reviewer: ```suggestion
If the previous command fails, make sure you have the latest version of Rust installed:
```
